x-pack/plugins/lens/public/editor_frame_service/editor_frame/frame_layout.tsx

```diff
@@ -18,67 +18,87 @@ export interface FrameLayoutProps {
   configPanel?: React.ReactNode;
   suggestionsPanel?: React.ReactNode;
   workspacePanel?: React.ReactNode;
+  bannerMessages?: React.ReactNode;
 }
 
 export function FrameLayout(props: FrameLayoutProps) {
   const isFullscreen = useLensSelector(selectIsFullscreenDatasource);
 
   return (
-    <EuiPage
-      className={classNames('lnsFrameLayout', {
-        'lnsFrameLayout-isFullscreen': isFullscreen,
-      })}
-    >
-      <EuiPageBody
-        restrictWidth={false}
-        className="lnsFrameLayout__pageContent"
-        aria-labelledby="lns_ChartTitle"
-      >
-        <section
-          className={classNames('lnsFrameLayout__sidebar lnsFrameLayout__sidebar--left', {})}
-          aria-labelledby="dataPanelId"
-        >
-          <EuiScreenReaderOnly>
-            <h2 id="dataPanelId">
-              {i18n.translate('xpack.lens.section.dataPanelLabel', {
-                defaultMessage: 'Data panel',
-              })}
-            </h2>
-          </EuiScreenReaderOnly>
-          {props.dataPanel}
-        </section>
-        <section
-          className={classNames('lnsFrameLayout__pageBody', {
-            'lnsFrameLayout__pageBody-isFullscreen': isFullscreen,
+    <EuiFlexGroup direction="column" responsive={false} gutterSize="none" alignItems="stretch">
+      {props.bannerMessages ? (
+        <EuiFlexItem grow={false}>
+          <section aria-labelledby="bannerMessagesId">
+            <EuiScreenReaderOnly>
+              <h2 id="bannerMessagesId">
+                {i18n.translate('xpack.lens.section.bannerMessages', {
+                  defaultMessage: 'Banner messages',
```

Comment:
Assuming the attempt here is to create a container that has the ability to house multiple deprecation messages and we don't wish to house them in the existing Lens warning popover, we should probably be more specific in our screenreader heading here. Perhaps `Deprecation warnings` instead of `Banner messages`.

Reply:
This is a common component that can be used not only for `Deprecation Messages` but also for example `Info Messages`, etc.  @flash1293 does it make sense to rename or not?

Reply:
As we only use it for deprecation at the moment I think it's OK to be specific here.